is.js

```diff
@@ -332,7 +332,7 @@
         creditCard: /^(?:(4[0-9]{12}(?:[0-9]{3})?)|(5[1-5][0-9]{14})|(6(?:011|5[0-9]{2})[0-9]{12})|(3[47][0-9]{13})|(3(?:0[0-5]|[68][0-9])[0-9]{11})|((?:2131|1800|35[0-9]{3})[0-9]{11}))$/,
         alphaNumeric: /^[A-Za-z0-9]+$/,
         timeString: /^(2[0-3]|[01]?[0-9]):([0-5]?[0-9]):([0-5]?[0-9])$/,
-        dateString: /^(1[0-2]|0?[1-9])\/(3[01]|[12][0-9]|0?[1-9])\/(?:[0-9]{2})?[0-9]{2}$/,
+        dateString: /^(1[0-2]|0?[1-9])[\/-](3[01]|[12][0-9]|0?[1-9])[\/-](?:[0-9]{2})?[0-9]{2}$/,
```

Comment:
You should make the first a capture group and use it as the second reference to avoid `'10-21/2012'` being valid.